[2024-06-26T11:18:58.792+0000] {scheduler_job_runner.py:769} ERROR - Executor reports task instance <TaskInstance: peduli_pintar_elt_all_dags.transform_dw_schema scheduled__2024-06-23T00:00:00+00:00 [queued]> finished (failed) although the task says it's queued. (Info: None) Was the task killed externally?
